saritasa_invocations/secrets.py

```diff
@@ -0,0 +1,49 @@
+import collections
+import collections.abc
+import re
+
+import invoke
+
+from . import k8s
+
+
+@invoke.task
```

Comment:
Let's either provide ability to specify list of args using CLI (see https://docs.pyinvoke.org/en/latest/concepts/invoking-tasks.html#iterable-flag-values), or remove `task` decorator from this function